src/graphql/main/schema.graphql

```diff
@@ -33,7 +33,45 @@ input AccountUpdateDefaultWalletIdInput {
 
 type AccountUpdateDefaultWalletIdPayload {
   account: ConsumerAccount
-  errors: [Error!]!
+  errors: [AppError!]!
```

Comment:
Is this type name change a breaking change?

Reply:
We resolved this by removing the unused error enums so that `code` could be set as a `GT.String` in the interface